subsys/nrf_security/src/drivers/cracen/silexpk/include/silexpk/ed25519.h

```diff
@@ -173,7 +173,7 @@ struct sx_pk_acq_req sx_pk_async_ed25519_sign_go(const struct sx_ed25519_dgst *k
  */
 void sx_async_ed25519_sign_end(sx_pk_req *req, struct sx_ed25519_v *sig_s);
 
-/** Verify an EDDSA signature (ED25519)
+/** Verify an EdDSA signature (ED25519)
```

Comment:
```suggestion
/** Verify an EdDSA signature (Ed25519)
```

if we fix the naming of the generic term, then the specialized version also needs to be fixed